models/note.model.js

```diff
@@ -0,0 +1,33 @@
+const mongoose = require("mongoose");
+const { Video } = require("./video.model");
+const { User } = require("./user.model");
+
+const noteSchema = new mongoose.Schema(
+  {
+    title: {
+      type: String,
+      required: true,
+    },
+    description: {
+      type: String,
+      required: true,
+    },
+    userId: {
+      type: mongoose.Schema.Types.ObjectId,
+      ref: "User",
+    },
+    videoId: {
+      type: String,
```

Comment:
this should be of type `mongoose.Schema.Types.ObjectId`

Reply:
Yes that is another way of doing it